tests/test_fio_workload.py

```diff
@@ -16,12 +17,45 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture()
+def pod_cleanup(request):
+    self = request.node.cls
+
+    def finalizer():
+        if hasattr(self, 'pod_obj'):
+            logger.info(f"Deleting pod {self.pod_obj.name}")
+            self.pod_obj.delete()
+        else:
+            logger.info(f"No pod_obj is present")
+    request.addfinalizer(finalizer)
+
+
+@pytest.fixture()
+def pvc_cleanup(request):
+    self = request.node.cls
+
+    def finalizer():
+        if hasattr(self, 'pvc_obj'):
+            self.pvc_obj.reload()
```

Comment:
@shylesh Is reload needed here, because pvc.get_pv_for_pvc() is also doing reload.